docs/src/components/Sidebar.astro

```diff
@@ -61,11 +61,34 @@ const sections = [
           .sort(({ frontmatter: { title } }) => (title === 'Main' ? -1 : 1))
           .map(({ url, frontmatter }) => (
             <li>
-              <a href={url || '/'}>{frontmatter.title}</a>
+              <a href={url || '/'} class="menu-link">
+                {frontmatter.title}
+              </a>
             </li>
           ))
       }
     </ul>
   </nav>
   {sections.map((section) => <NavSection section={section} />)}
 </aside>
+
+<style is:global>
```

Comment:
Why global? Try to avoid it.

Reply:
Because I needed to apply styles both on sidebar tags and `NavSections` tags. I tried to pass [class as a prop](https://docs.astro.build/en/guides/styling/#passing-a-class-to-a-child-component) but it didn't help, it needs to pass two classes. I moved styles to `tags.css` file for now. Do you know better ways?